packages/taiko-client/proposer/proposer.go

```diff
@@ -313,9 +313,15 @@ func (p *Proposer) ProposeTxLists(ctx context.Context, txLists []types.Transacti
 			return nil
 		}
 
-		if err := p.ProposeTxListPacaya(ctx, txLists); err != nil {
-			return err
+		// Make sure the tx list is not bigger than the maxBlocksPerBatch.
+		maxBlocksPerBatch, _ := p.protocolConfigs.MaxBlocksPerBatch()
```

Comment:
Let's check the error here, or u don't need to make this function return an error.

Reply:
No need to return error, and removed.